src/main/kotlin/net/maizegenetics/phgv2/utils/FileUtils.kt

```diff
@@ -0,0 +1,39 @@
+package net.maizegenetics.phgv2.utils
+
+import java.io.*
+import java.net.URL
+import java.util.zip.GZIPInputStream
+import java.util.zip.GZIPOutputStream
+
```

Comment:
Add file header comment (because they all should have one).  I realize the name of the file is pretty explanatory.

Reply:
Comments added.